### PR TITLE
implement upcoming event card

### LINKED ISSUE #9: Implement Upcoming Event Card
image

* Just implement one of the cards
* Use props
* Don't put on any page, only for testing purposes and screenshot purposes

### PR DESCRIPTION
**Pull request by sshakya03**
- Created event card component
- Closes #9

### CODE REVIEW

**Comment by sshakya03** [comment]
Screenshot 2024-07-05 at 9 30 27 PM
Screenshot 2024-07-05 at 9 30 48 PM

File: src/Components/events/Card.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+const Card = ({ month, day, hour, name, location, description }) => {
+  return (
+    <div className="rounded-2xl flex flex-row justify-center drop-shadow-2xl mx-auto w-6/12">
+      <div className="bg-cyan-500 flex flex-col justify-center rounded-l-2xl text-[#ffffff] py-4 w-1/5">
+        <div className="font-russo font-medium mt-3 text-center md:text-4xl">
+          {month}
+        </div>
```
Comment (khandrew1) [inline src/Components/events/Card.js]: use p tags instead of divs for the text, reserve divs for like boxing components

**Comment by khandrew1** [comment]
In terms of fixing the colors, merge in the dev branch and use the custom colors in the tailwind config

**Comment by sshakya03** [comment]
Screenshot 2024-07-08 at 2 40 07 PM

updated with custom color

**Review by khandrew1 — APPROVED** [review]

**Pull request merged**